va/va_enc_mpeg2.h

```diff
@@ -287,6 +287,17 @@ typedef struct _VAEncSliceParameterBufferMPEG2 {
     uint32_t                va_reserved[VA_PADDING_LOW];
 } VAEncSliceParameterBufferMPEG2;
 
+typedef struct _VAEncMiscParameterExtensionDataSeqDisplayMPEG2
+{
+    uint8_t extension_start_code_identifier;
```

Comment:
Why to include the "extension_start_code_identifier" ? Seems redundant to me..

Reply:
extension_start_code_identifier is used to distinguish the start code of the extension, as the spec, there are sequence display extension, quant matrix extension,sequence salable extension etc.

Reply:
Table 6-2. extension_start_code_identifier codes.

Reply:
I know there is extension_start_code_identifier for all extensions, but my question is about the requirement of this filed in structure. It is very clearly mentioned as "ExtensionDataSeqDisplayMPEG2" so there only one possible predefined value for extension_start_code_identifier.  We don't really include the start_code(s) in any public structures, do we?

Reply:
yes, so there are two method to define the structure, 1. define two structure , first one is structure Mpeg2SeqExtension{ uint8_t extension_start_code_identifier; uint8_t data[]} another structure is typedef structure ExtensionDisplay{...). 2. define extension_xxx directly with first byte is extension_start_code_identifier. using method 1 will let there are multiple data[] definition in the VAEncMiscParameterBuffer ( Misc type, Misc data, Extension type, extension data).   so I choose second one. I will add comments about the default value of extension_start_code_identifier.